nemo_text_processing/inverse_text_normalization/en/taggers/tokenize_and_classify.py

```diff
@@ -58,15 +61,23 @@ def __init__(
         cache_dir: str = None,
         overwrite_cache: bool = False,
         whitelist: str = None,
+        profane_words: str = None,
+        filter_profanity: bool = True,
```

Comment:
Makes sense. I have made `False` as the default value for  `filter_profanity`.

In a corporate world, it looked very much needed in all cases, but it may not be the case always for all use cases.